docs/data-management.rst

```diff
@@ -508,6 +514,17 @@ Step-by-Step
     duplicates, the link might be deleted during the subsequent
     move of that duplicate to another group.
 
+.. _Shares:
+
+Shares (discontinued feature)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously created Shares can still be viewed in the ``Shares`` tab |image6| above the left-hand side pane of OMERO.web. Nevertheless, the Shares feature is discontinued. **It is not posssible to use the** ``World`` icon |image7| above the left-hand pane in OMERO.web to create new Shares anymore. Shares have been deprecated in OMERO 5.3.0 and are discontinued in OMERO.web xxx and later. You need to use the OMERO standard permissions to share Images, by :ref:`moving the data<Movedowners>` into the appropriate group. If you also want to retain the Images in their current group, you can first duplicate them as described in :ref:`Duplicate feature<Duplicatecli>`. Nevertheless, this :ref:`Duplicate<Duplicatecli>` and :ref:`Move<Movedowners>` workflow has following limitations:
+
+ - :ref:`Duplicate<Duplicatecli>` is only available on the Command Line Interface (CLI)
+ - You need to already be in a non-private group with the user you wish to share with
+ - You will also be sharing the data with all the other members of that group
```

Comment:
At least on https://github.com/pwalczysko/omero-guide-introduction/blob/share/docs/data-management.rst page, these haven't formatted correctly. They have an extra space before ` -`

Reply:
the rendering on https://github.com/pwalczysko/omero-guide-introduction/blob/share/docs/data-management.rst is not relevant.  Try to complare the rendering of an existing link (not changed in this PR) on github with the correct rendering on the released guide in 

The rendering of an existing link (not changed in this PR) on github
![Screenshot 2021-03-08 at 12 46 29](https://user-images.githubusercontent.com/2478303/110323348-6527c480-800c-11eb-9e6d-58791ee92fb9.png)

This renders correctly in https://omero-guides.readthedocs.io/projects/introduction/en/latest/data-management.html?highlight=%22Duplicate%20feature%22#manage-images-in-datasets-projects
![Screenshot 2021-03-08 at 12 44 51](https://user-images.githubusercontent.com/2478303/110323415-7ec90c00-800c-11eb-97cb-1857b93ea35a.png)

The markdown on the line matches to what was there before in places such as screenshotted above. It also renders correctly if you do ``make clean html`` with local sphinx.